python/legate_dataframe/lib/core/column.pyx

```diff
@@ -106,6 +109,29 @@ cdef class LogicalColumn:
                 "from_cudf() only supports cudf columns and device scalars."
             )
 
+    @staticmethod
+    def from_arrow(array) -> LogicalColumn:
+        """Create a logical column from a local arrow array.
+
+        This call blocks the client's control flow and scatter
```

Comment:
I guess technically, we'll have a local copy on all nodes due to control replication :).

Not sure, how to adapt, just delete the second part or is it more right than it seems to me?